app/move/views/confirmation.njk

```diff
@@ -1,13 +1,14 @@
 {% extends "layouts/base.njk" %}
 
 {% block customGtagConfig %}
-  gtag('set', {'page_title': 'Move requested'});
+  gtag('set', {'page_title': '{{ "Person added to allocation" if move.allocation else "Move " + move.status }}' });
```

Comment:
Just a thought. Could we actually use different templates for move and assign? As long as we never need another variant, I suppose this is cool

```
{% block customGtagConfig %}
  gtag('set', {'page_title': '{% block CustomGtagContent %}{% endblock %}'});
{% endblock %}
...
{% block CustomGtagContent %} Person added to allocation{% endblock %}
...
{% block CustomGtagContent %} Person added to allocation{% endblock %}
```

Reply:
I suppose I was trying to keep them combined so that we keep consistency between confirmation pages.

So any of the actions that occur on a move use this same confirmation so felt better to maintain having one approach. The display and layout is all the same but because they were maintained separately we ended up with them looking differently.

Reply:
Sorry, to be clear. I meant have the templates share the same base confirmation template and just override the bits as needed.  eg.  how the base govuk template allows the page title to be overridden through the `pageTitle` block. If there are only 2 variants, it's arguable that it's clearer to just have if statements for the 2 cases.